.github/workflows/critical-issue.yml

```diff
@@ -29,7 +29,7 @@ jobs:
                   "type": "section",
                   "text": {
                     "type": "mrkdwn",
-                    "text": "Please check the issue: ${{ github.event.issue.url }}\n \\cc @zeebe-process-automation-team @zeebe-distributed-platform-team "
+                    "text": "Please check the issue: ${{ github.event.issue.html_url }}\n \\cc @zeebe-process-automation-team @zeebe-distributed-platform-team "
```

Comment:
@nicpuppa Where did you find this? Are you sure this property exists? Because I don't see it in the docs.

Reply:
I suppose it'll work fine, as I see this is returned in an issue object. I'm probably just confused by GitHub docs 😄

Reply:
GitHub docs are a mess. This property should be available into the issue object